src/Devices/Ethernet_TC.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Ethernet_TC.h"
+using namespace std;
+
+TC_ethernet *TC_ethernet::_instance = nullptr;
+
+TC_ethernet::TC_ethernet() {
+  pinMode(4, OUTPUT);
+  digitalWrite(4, HIGH);
+  if (Ethernet.begin(mac) == 0) {
+    Serial.println(F("Failed to configure Ethernet using DHCP"));
+    Ethernet.begin(mac, defaultIP);
+    IP = defaultIP;
+  }
+
+  defaultIP = IPAddress(192, 168, 1, 2);
+  time_serverIP = IPAddress(132, 163, 97, 1);
```

Comment:
Does anyone use this?

Reply:
I couldn't find any references in the original implementation for the Time_server. But I'll remove it for now (since I'm not using it), and add it back in later if it's needed.